fetch_printer_status.py

```diff
@@ -0,0 +1,16 @@
+import json
+from data.printer_config import PRINTERS
+import fdmprinting.archive_retrieve as ar
+
+def get_printer_status():
+
+    for p in PRINTERS:
+        printer = PRINTERS[p]
+        printer_object = ar.get_status_from_mqtt(printer, printer["name"])
+        clean_json = ar.clean_raw_json(printer_object["raw_json"])
+
+        with open("./"+ printer["name"] + ".json", "w") as my_file:
+            my_file.write(json.dumps(clean_json, indent=4))
+
+if __name__ == "__main__":
+    get_printer_status()
```

Comment:
I think we should include the outer JSON as well as add a wee bit of status output. We can add PEP 8 compliant line breaks. Finally, we can get away with one `import`:

```suggestion
import fdmprinting.archive_retrieve as ar


def get_printer_status():
    for id in ar.PRINTERS:
        printer = ar.PRINTERS[id]
        file = printer["name"] + ".json"
        print("Writing " + file)
        printer_object = ar.get_status_from_mqtt(printer, id)
        printer_object["raw_json"] = ar.clean_raw_json(printer_object["raw_json"])
        with open(file, "w") as my_file:
            my_file.write(ar.json.dumps(printer_object, indent=4))
    print("Done")


if __name__ == "__main__":
    get_printer_status()
```